src/js/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -76,6 +76,7 @@ export class VeteranBenefitSummaryLetter extends React.Component {
                 name={key}
                 type="checkbox"
                 onChange={this.handleChange}/>
+              <label/>
```

Comment:
I haven't checked, but I'm wondering if this could cause accessibility issues.  It could be that the `aria-labelledby` attribute above overrides this blank label being associated with the input, but we should probably verify manually.

Suzanne wrote up some Voice Over instructions in case you haven't used it before: https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Work%20Practices/Accessibility%20and%20508/Mac_VoiceOver_guide.md
I guess I'd just check that it doesn't announce the extraneous blank label and confuse the user.

Reply:
Yeah, was wondering about that but seemed to pass accessibility checks. I'll verify manually and share an update here shortly - thanks for the instructions.

Reply:
Thanks for checking!